app/views/api_users/_form.html.haml

```diff
@@ -3,24 +3,16 @@
     - fieldset.radio_input :yes, panel_id: 'is_government_panel'
     - fieldset.radio_input :no, panel_id: 'not_government_panel'
   - f.revealing_panel :is_government_panel do |panel|
-    = f.collection_select :department, government_orgs_local_authorities, :first, :last, { include_blank: true }
     = f.text_field :email_gov, type: 'email'
+    = f.radio_button_fieldset :contactable, inline: true
```

Comment:
if the user does not select an option for this we need to show a validation error. It is not optional.